main.go

```diff
@@ -190,43 +212,56 @@ func main() {
 	}
 	filePath := os.Args[1]
 	debug("## Loading " + filePath)
+	
+	var test Test
+	var summary Summary
+	var subsetPartition map[int][]int
+
+	readTestFile(filePath, &test)
+	validate(&test, &subsetPartition)
```

Comment:
To my like, this would be more standard:

```
subsetPartition, err := partition(test.Config)
if !validate(test, subsetPartition) -> error
summary := RunTests(test, subsetPartition)
```